sklearn/linear_model/_glm/glm.py

```diff
@@ -430,6 +430,19 @@ class PoissonRegressor(GeneralizedLinearRegressor):
 
     n_iter_ : int
         Actual number of iterations used in the solver.
+
+    Examples
+    ----------
+    >>> from sklearn import linear_model
+    >>> clf = linear_model.PoissonRegressor(alpha=0.95)
+    >>> X = [[0, 0], [1, 1], [2, 2]]
+    >>> y = [0, 1, 2]
+    >>> clf.fit(X, y)
+    PoissonRegressor(alpha=0.95)
+    >>> print(clf.coef_)
+    [0.30189415 0.30189415]
```

Comment:
```suggestion
    [0.301... 0.301...]
```

Reply:
Thanks. I applied this format (also other results)